src/platform/imx95/include/platform/lib/clk.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2024 NXP
+ */
+
+#ifdef __SOF_LIB_CLK_H__
+
+#ifndef __PLATFORM_LIB_CLK_H__
+#define __PLATFORM_LIB_CLK_H__
+
+#define CLK_MAX_CPU_HZ 800000000
+#define CPU_DEFAULT_IDX 0
+#define NUM_CPU_FREQ 1
+#define NUM_CLOCKS 1
+
```

Comment:
@LaurentiuM1234 I think @kv2019i got rid of a lot of these last week since they should come from Zephyr - can you check.

Reply:
hm...double-checked and couldn't find anything else to drop w/o breaking the build... @kv2019i anything obvious that I'm missing?

Reply:
ok, I'm probably wrong, probably in process of being removed.

Reply:
Ack @LaurentiuM1234  @lgirdwood platform clk.h is still needed. We have a Zephyr version of clk.h that still depends on it. OTOH, I did a quick look and it seems this could be cleaned up further as well as the usage of clk.h is not much in app code -> https://github.com/thesofproject/sof/issues/9541